---
layout: default
---
## Bio:
I am a PhD candidate at Stanford University, advised by Anshul Kundaje in Computer Science & Genetics, and co-advised by Brian Hie in Chemical Engineering. My current research focuses on modeling and engineering molecular-scale biology using deep learning methods.

For my undergraduate, I studied Chemical Engineering at Columbia University, with research in electrocatalytic reduction of CO2 using non-toxic, earth-abundant metals with Ponisseril Somasundaran.  
For my Master's, I studied Biotechnology at the University of Cambridge, with research in engineering bacterial endospores for DNA data storage with Graham Christie.
For my PhD, I am currently a part of the Chemical Engineering Department.

## Projects:
#### JASPAR 2026: expansion of transcription factor binding profiles and integration of deep learning models
![Figure: JASPAR 2026: Deep learning collection.](/assets/img/project_jaspar.jpg)
___Figure:___ _JASPAR 2026: Deep learning collection: Characterizes TFâ€“DNA interactions with 1,259 BPNet models trained on Homo sapiens ENCODE chromatin immunoprecipitation followed by sequencing (ChIP-seq) datasets from 240 TFs and interpreted to reveal predictive motif patterns for the models. The motifs associated with the same TF were clustered to provide a summary of the binding properties, resulting in 240 primary and 113 alternative motif patterns in the DL collection. The top panel illustrates the comprehensive workflow. The bottom panels present screenshots of the TF summary profile page (left) and the model page (right)._

- __Publication__: [https://doi.org/10.1093/nar/gkaf1209](https://doi.org/10.1093/nar/gkaf1209) 
- __JASPAR: Deep Learning Collection__: [https://jaspar.elixir.no/collection/deep-learning/](https://jaspar.elixir.no/collection/deep-learning/) 

---
#### A unified lexicon of predictive DNA sequence motifs from ENCODE transcription factor binding and chromatin accessibility assays
![Figure: MotifCompendium: A GPU-accelerated Python package for clustering, annotating, and managing motifs, at scale.](/assets/img/project_mc_main.png)
![Figure: ENCODE MotifCompendium: Pipeline process.](/assets/img/project_mc_pipeline.png)
___Figure:___ _MotifCompendium: A GPU-accelerated Python package for clustering, annotating, and managing motifs, at scale. Example process collapsing FOX dimer motif, starting from TF ChIP-seq BPNet-derived motifs into ENCODE TF MotifCompendium pattern. The process was completed on the ENCODE Project, across 19,739 TF ChIP-seq BPNet-derived motifs, resulting in 1,921 unique motif patterns._

- __Technical Note__: [https://doi.org/10.5281/zenodo.17123347](https://doi.org/10.5281/zenodo.17123347) 
- __GitHub__: [https://github.com/kundajelab/motifcompendium](https://github.com/kundajelab/motifcompendium) 
- __ENCODE Project__: [https://www.encodeproject.org/](https://www.encodeproject.org/) 

---
#### Designing _de novo_ bacterial toxin-antitoxins using a generative genomic foundation model
![Figure: Strategy for de novo design of Type II toxin-antitoxins using a genomic foundation model.](/assets/img/project_tat.png)
___Figure:___ _Strategy for de novo design of Type II toxin-antitoxins using a genomic foundation model. (a) Mechanism of Type II toxin-antitoxins. (b) High-level overview of the strategy for designing novel Type II toxin-antitoxins to expand the existing repertoire._

- __GitHub__: [https://github.com/chang-m-yun/CS273B_TA_Evo](https://github.com/chang-m-yun/CS273B_TA_Evo)  

---
#### Predicting post-transcriptional ADAR activity using a language model
![Figure: Adenosine Deadmidase acting on RNA (ADAR) activity.](/assets/img/project_adar.png)  
___Figure:___ _Adenosine Deadmidase acting on RNA (ADAR) activity_

- __GitHub__: [https://github.com/chang-m-yun/CS229_final_project/](https://github.com/chang-m-yun/CS229_final_project/)
